### PR TITLE
feat: autogenerate typescript, python, java SDKs with swagger

### PR DESCRIPTION
**Pull request by asdfryan**
The SDKs were generated via https://swagger.io/tools/swagger-codegen/

```
swagger-codegen generate -i openapi/crm/openapi.bundle.json -l java -o packages/sdk/java
```

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -4,3 +4,4 @@
 /.pnp.*                         binary linguist-generated
 /packages/schemas/gen/**/*      linguist-generated
 openapi/**/openapi.bundle.json  linguist-generated
+/packages/sdk/**/*              linguist-generated
```
Comment (lucasmarshall) [inline .gitattributes:7]: Can we omit the README?

Also can we add a `packages/sdk/package.json` with a "generate" script that runs the command in the README?

Comment (asdfryan) [reply .gitattributes:7]: Done.

File: openapi/common/components/parameters/customer-id.yaml [hunk 2]
```diff
@@ -2,7 +2,6 @@ name: customer-id
 in: header
 schema:
   type: string
-  format: uuid
```
Comment (asdfryan) [inline openapi/common/components/parameters/customer-id.yaml:5]: I needed to get rid of this so the codegen does not escape the input customerId.

File: openapi/crm/openapi.yaml [hunk 3]
```diff
@@ -20,7 +20,7 @@ externalDocs:
   description: Try out our API in Postman!
   url: 'https://www.postman.com/supaglue/workspace/supaglue-public/overview'
 servers:
-  - url: 'https://localhost:8080/crm/v1'
+  - url: 'http://localhost:8080/crm/v1'
```
Comment (asdfryan) [inline openapi/crm/openapi.yaml:23]: To fix the default BASE_URL in the generated client

**Review by albertyfwu — APPROVED** [review]

**Review by tomkit — APPROVED** [review]

**Review by tomkit — APPROVED** [review]

**Pull request merged**